docs/demo.md

```diff
@@ -101,7 +101,7 @@ total balance = 0.14974828 BTC
 
 * Teleport detects which network it's on by asking the Bitcoin node it's connected to via json-rpc. So to switch between networks like `regtest`, signet, testnet or mainnet (for the brave), make sure the RPC host and port are correct in `src/lib.rs`.
 
-* You will need Tor running on the same machine, then open the file `src/directory_servers.rs` and make sure the const `TOR_ADDR` has the correct Tor port.
+* You will need Tor running on the same machine, then open the file `src/directory_servers.rs` and make sure the const `TOR_SOCKS_ADDR` has the correct Tor port.
```

Comment:
As we are spawning our maker and directory tor instance and on non general port. So, it might not clash , but if it does we should look onto some other port. Can be thought of once we finalize this. I am thinking of adding docs once we are sure with our approach.